content/projects/metabake.md

```diff
@@ -1,18 +1,19 @@
 ---
 title: Metabake
-repo: metabake/mbake-CLI-for-Metabake
-homepage: http://www.metabake.net
+repo: metabake/MetaBake-Examples
+homepage: http://mBake.org
 language: JavaScript
 license: MIT
 templates: Pug and Markdown
-description: Low-code productivity for programmers via generators for Pug, Markdown and much more; including dynamic data binding.
+description: MetaBake is the extensible open source low-code productivity tool for programmers; including dynamic apps and data binding.
 ---
 
 
-#### Metabake.org is a low-code productivity for application developers via static generation; with Pug, Markdown and more; including dynamic data binding. *Some developers implement applications faster than others.*
+## MetaBake is the extensible open source low-code productivity tool for programmers; including dynamic apps and data binding.
 
+#### MetaBake is the extensible open source low-code productivity tool for programmers, via static generation; with Pug, Markdown and more; including dynamic apps and data binding. *'All. My. Friends. Know a low-coder.'*
```

Comment:
This seems to be a near duplicate of the line above it.

Reply:
fixed in add to PR if I did it correctly.